cc/selfplay.cc

```diff
@@ -375,16 +391,22 @@ class SelfPlayer {
                        << player->options().virtual_losses << ")";
           MG_LOG(INFO) << root->CalculateTreeStats().ToString();
 
-          MG_LOG(INFO) << root->position.ToPrettyString(use_ansi_colors);
-          MG_LOG(INFO) << "Move: " << position.n()
-                       << " Captures X: " << position.num_captures()[0]
-                       << " O: " << position.num_captures()[1];
-          MG_LOG(INFO) << root->Describe();
+          if (!fastplay) {
```

Comment:
the prevalence of `!fastplay` suggests to me that slowplay is the right way to think about it?

Reply:
agreed.

Reply:
While that makes sense, "slowplay" is essentially the defaults.  So these flags are specifying the other behavior.